openghg/data/source_format_data.csv

```diff
@@ -10,6 +10,17 @@ SIO,medusa,AGAGE,,GCWERKS
 JFJ,medusa,AGAGE,,GCWERKS
 CMN,medusa,AGAGE,,GCWERKS
 ZEP,medusa,AGAGE,,GCWERKS
+MHD,medusa,AGAGE,,AGAGE
```

Comment:
As discussed, I have a small update suggestion for this. Since there are now multiple types for a few different sites, would be useful to add another column to just distinguish these. I was thinking something like a "File_type" column with "txt" for the "GCWERKS" entries and "nc" or "netcdf" for the "AGAGE" entries. Can also be left blank for other types if needed.